src/common/meta/src/key/table_route.rs

```diff
@@ -481,7 +481,7 @@ impl TableRouteStorage {
     /// Builds a update table route transaction,
     /// it expected the remote value equals the `current_table_route_value`.
     /// It retrieves the latest value if the comparing failed.
-    pub(crate) fn build_update_txn(
+    pub fn build_update_txn(
```

Comment:
What is the reason for making it public?

Reply:
had to use it in some testing codes. i will change it back after some refactor